### ISSUE TITLE
Transparent and inherit fallback color

### ISSUE THREAD

**Issue by bbrdaric** [CSSLint/csslint#324]
Hi,

if I have a transparent background-color, I get warning about fallback color.
As there is no hex code for transparency, it's the only way this could be done.

IMO, transparent and inherit properties should be allowed, as defined in
http://www.w3.org/TR/CSS2/colors.html#propdef-background-color

Is something wrong with that? I just tested transparent and inherit properties as fallback for rgba in IE8 and that works as expected.

``` CSS
.el{
  background-color: transparent;
  background-color:rgba(0,0,0,0.5);
}
```

> Fallback background-color (hex or RGB) should precede RGBA background-color.
> background-color:rgba(0,0,0,0.5);

Regards,
Boris.
